tasks.py

```diff
@@ -686,3 +686,10 @@ def generate_self_signed(domain):
         ' -keyout {0}.key -out {0}.crt'
     ).format(domain)
     run(cmd)
+
+@task
+def update_citation_styles():
+    print("Not implemented. Do the following manually:")
```

Comment:
Is there a barrier to implementing this? Can you not use the `run()` function for each of these commands?